packages/components/src/Chip/Chip.css

```
@@ -23,7 +23,8 @@
 }
 
 .chip:focus,
-.chip:focus-visible {
+.chip:focus-visible,
+.clickableSuffix:focus-visible {
   outline: none;
```

```suggestion
  outline: transparent;
```

I know you didn't introduce this, but only manipulating the outline color keeps the focus states compatible with Windows hi-constrast mode